waffle_hub/dataset/dataset.py

```diff
@@ -1398,6 +1391,9 @@ def add_images(self, images: list[Image]):
         Args:
             images (list[Image]): list of "Image"s
         """
+        self.__dict__.pop("images", None)
```

Comment:
Is it the best way to do it?
It seems quite odd.

Reply:
> Is it the best way to do it? It seems quite odd.

I haven't found any method other than this one.